test_conformance/spirv_new/main.cpp

```diff
@@ -81,7 +81,12 @@ void spirvTestsRegistry::addTestClass(baseTestClass *test, const char *testName)
     test_definition testDef;
     testDef.func = test->getFunction();
     testDef.name = testName;
-    testDef.min_version = Version(2, 1);
+    if (strstr(testName, "op_spec_constant_")) {
```

Comment:
This is a nasty hack :). We will need to change the test registration logic to be able to pass an optional min version. I'd like to reuse some of that registration logic a bit more widely to reduce the amount of boilerplate so I'd like to keep it reasonably clean. I suggest doing this as a separate pre-requisite patch.

Reply:
Ok thanks. Yes it is nasty but works :D.